### ISSUE TITLE
agena view: span week doesn't show tasks from this week

### ISSUE THREAD

**Issue by tbm** [dhruvasagar/vim-dotoo#79]
With the "day" view I get:

```
Date: Thursday 09 July 2020, Span: day
       test:         50y 202d ago: TODO Test 1
       test:         50y 202d ago: TODO Test 2                                        
```

But when I switch to "week" or "month" view I get:

```
Date: Monday 06 July 2020, Span: week
   No pending tasks!
```

Why does the week span show nothing when there are clearly tasks this week (today).

Example file:

```
* Test
*** TODO Test 1
SCHEDULED: <2020-07-09 Thu 12:00> DEADLINE: <2020-07-09 Thu 17:00>

    Test 1

** TODO Test 2
SCHEDULED: <2020-07-10 Fri 12:00> DEADLINE: <2020-07-10 Fri 17:00>
```

**Comment by tbm** [comment]
The problem is that `SCHEDULED` and `DEADLINE` were on the same line. that seems to confuse vim-dotoo. Note the "50y 202d ago".

I believe this is valid in Org mode though. At least Orgzly produces such files.

**Comment by tbm** [comment]
This is related to #80

**Comment by tbm** [comment]
> I believe this is valid in Org mode though. At least Orgzly produces such files.

I believe it's mandated in Org, see #80

**Comment by Gavinok** [comment]
@tbm I've had the same issue. However since as far as I know vim-dotoo does not support using both SCHEDULED and DEADLINE in the same heading I just use one until the feature can be added.

**Comment by dhruvasagar** [comment]
Yes I have never tested / used this feature so I am not exactly aware on the exact behavior associated with having both. However, logically, it does make sense to allow both. Will check the metadata parser.

**Comment by dhruvasagar** [comment]
Also note, in general vim-dotoo doesn't do a good job distinguishing between a scheduled date vs a deadline date, that's also a known issue and something that should be fixed sooner rather than later.

**Comment by dhruvasagar** [comment]
For folks interested, I have pushed a new update (Version 0.12.0) where i've significantly improved the Agenda View so it distinguishes between Scheduled headlines vs Headliens with a deadline properly. I've kept the logic to be close to how I understand emacs org-mode does.

I still haven't addressed the issue of supporting both a deadline & a schedule on the same headline. I will work on the separately in the coming releases.
